src/Tabs.jsx

```diff
@@ -63,9 +63,15 @@ export class Tabs extends React.Component {
 	}
 }
 Tabs.propTypes = {
-	children: React.PropTypes.arrayOf(
-		React.PropTypes.instanceOf(TabsTab)
-	).isRequired,
+	children: (props, propName, componentName) => {
+		let error;
+		React.Children.forEach(props[propName], child => {
+			if (typeof child === 'undefined' || child.type !== TabsTab) {
```

Comment:
I recommend separating out your two `child` validation conditions - use [`React.Children.count`](https://facebook.github.io/react/docs/react-api.html#react.children.count) to return an Error about passing in _no_ children, and use the `forEach` that you have here to check the type of the children.

The separation will make it a little more clear what is needed and simplify the code a bit

```js
const children = props[propName];
if (!React.Children.count(children)) {
  return new Error(...);
}
```